BULK VALIDATION description-missing (#1427)
dotnet/docs-desktop: dotnet-desktop-guide/framework/winforms/advanced/working-with-images-bitmaps-icons-and-metafiles.md

---
title: "Working with Images, Bitmaps, Icons, and Metafiles"

> description: Learn more about working with images, bitmaps, icons, and metafiles in this article. Learn more about working with raster and vector images.

Reviewer: ```suggestion
description: Learn about which articles you can read to learn more about working with images, bitmaps, icons, and metafiles.
```

Author: Addressed.
